src/pkg/packages.builds

```diff
@@ -7,10 +7,18 @@
          they will be built during the source build phase -->
     <AdditionalProperties>BuildPackageLibraryReferences=false</AdditionalProperties>
   </PropertyGroup>
+  
+  <!-- required to build the projects in their specified order -->
+  <!-- This ensures the MakeAppx target in Microsoft.NET.CoreRuntime runs before
```

Comment:
This make the projects build in serial but doesn't guarantee the order, we still need to make sure the UWPCoreRuntime.depproj gets built before the sdk packaging does.